src/main/java/com/thoughtworks/gauge/processor/ValidateStepProcessor.java

```diff
@@ -39,20 +40,37 @@ public ValidateStepProcessor(ClassInstanceManager instanceManager) {
 
     private StepValidateResponse validateStep(Messages.StepValidateRequest stepValidateRequest) {
         Set<Method> methodImplementations = StepRegistry.getAll(stepValidateRequest.getStepText());
+
+        final StringBuilder suggestion = new StringBuilder(String.format("\n\nSuggestion : \n@Step(\"%s\")\n", stepValidateRequest.getStepValue().getParameterizedStepValue()));
+        suggestion.append(String.format("public void implementation(%s){\n", getParamList(stepValidateRequest.getStepValue().getParametersList())));
+        suggestion.append("}\n");
```

Comment:
might be worth adding a comment line so that the output looks something like this:

```
Suggestions:
@Step(<STEP TEXT>)
public void implementation(<args>){
    // your code here...
}
```